bin/find-unused-state-action-types.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+grep -r "export const [A-Z_]* =" client/state/action-types.ts | \
```

Comment:
Should we have a check to ensure the action types file exists before doing anything else? Could be helpful if someone removes or moves the file without knowing about this script.